.expeditor/buildkite/cypress.sh

```diff
@@ -25,9 +25,7 @@ do
   export CYPRESS_BASE_URL="https://$instance"
   export CYPRESS_RECORD_KEY="$CYPRESS_RECORD_KEY"
 
-  npm install # get dependencies defined in e2e/package.json
```

Comment:
Why is this step being deleted? I am surprised it would still work without this.

Reply:
This step is redundant. I think it was a stop-gap solution while releng updated the base docker image to install cypress and its dependencies.

Testing with the docker container locally, it passed without `npm install`.

```
# check out this branch
# run from automate/
docker run -it --rm --volume $(PWD):/workdir --workdir /workdir chefes/releng-base bash
export CHANNEL=acceptance
.expeditor/buildkite/cypress.sh

# should see output like this:
true
true
false
--- Executing Cypress tests against a2-perf-test-single-local-inplace-upgrade-acceptance.cd.chef.co
> automate-e2e@1.0.0 cypress:run /workdir/e2e
> cypress run
It looks like this is your first time using Cypress: 4.1.0
  ✔  Verified Cypress! /root/.cache/Cypress/4.1.0/Cypress
Opening Cypress...
```